pkg/loop/internal/pb/contract_reader.proto

```diff
@@ -46,7 +48,7 @@ message UnbindRequest {
 
 // GetLatestValueReply has return arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ContractReader.GetLatestValue].
 message GetLatestValueReply {
-  VersionedBytes ret_val = 1;
+    VersionedBytes versioned_bytes = 1;
```

Comment:
I would revert changes that aren't 100% necessary.  A rename could break things in core or other repos.

Reply:
sorted